src/network/connection.cpp

```diff
@@ -62,18 +62,29 @@ namespace con
 
 #define PING_TIMEOUT 5.0
 
-BufferedPacket makePacket(Address &address, const SharedBuffer<u8> &data,
+u16 BufferedPacket::getSeqnum() const
+{
+	if (size() < BASE_HEADER_SIZE + 3) {
+		//std::cout << (int)data[0xFFFFFFFFF] << std::endl;
+		throw ConnectionException("Packet too short to contain a seqnum");
+	}
```

Comment:
The check is here btw:
https://github.com/minetest/minetest/blob/adead3fd8392354dfd5abbbccb248e422ebad4db/src/network/connection.cpp#L262-L266

So I think this would be fine:
```suggestion
	if (size() < BASE_HEADER_SIZE + 3)
		return 0; // should never happen
```

Reply:
Yes, this exception is not triggered within the current code, but I felt like adding it would aid troubleshooting in the future. Nonetheless, I will apply this suggestion because I don't mind much about this.